packages/grid/_modules_/grid/components/columnSelection/GridCellCheckboxRenderer.tsx

```diff
@@ -45,6 +48,10 @@ export const GridCellCheckboxForwardRef = React.forwardRef<HTMLInputElement, Gri
 
     const CheckboxComponent = apiRef?.current.components.Checkbox!;
 
+    const isSelectable = options.isRowSelectable
+      ? options.isRowSelectable(apiRef!.current.getRowParams(id))
+      : true;
```

Comment:
A small thing but looks nicer 😇

```suggestion
    const isSelectable = options.isRowSelectable || options.isRowSelectable(apiRef!.current.getRowParams(id))
```

Reply:
With the OR operator, `options.isRowSelectable` will never be called if it's defined.

Reply:
```
const isSelectable = options.isRowSelectable== null || options.isRowSelectable(apiRef!.current.getRowParams(id))
```